src/EnergyPlus/EMSManager.cc

```diff
@@ -2012,9 +2012,9 @@ namespace EMSManager {
 			if ( ! Surface( SurfNum ).HeatTransSurf ) continue;
 			if ( ! ( Surface( SurfNum ).ExtBoundCond == ExternalEnvironment ) ) continue;
 
-			SetupEMSActuator( "Surface", Surface( SurfNum ).Name, "Outdoor Air Dryblub Temperature", "[C]", Surface( SurfNum ).OutDryBulbTempEMSOverrideOn, Surface( SurfNum ).OutDryBulbTempEMSOverrideValue );
+			SetupEMSActuator( "Surface", Surface( SurfNum ).Name, "Outdoor Air Drybulb Temperature", "[C]", Surface( SurfNum ).OutDryBulbTempEMSOverrideOn, Surface( SurfNum ).OutDryBulbTempEMSOverrideValue );
```

Comment:
@EnergyArchmage @mjwitte @mitchute 

Do you anticipate this needing a transition rule?